reconcile/utils/sentry_client.py

```diff
@@ -27,10 +28,35 @@ def _do_sentry_api_call_(self, method, path, slugs, payload=None):
         response.raise_for_status()
 
         try:
-            return response.json()
+            all_results = response.json()
         except json.decoder.JSONDecodeError:
             return
 
+        # there may be more pages if the response contains a link header
```

Comment:
Thank you for adding a note here!

Reply:
Perhaps add an example as well for easier reading?

Reply:
Even better if we add a link to the sentry API documentation here, if available

Reply:
this is the example (right here below):
https://github.com/app-sre/qontract-reconcile/pull/1303/files#diff-c4b8fdf06e29c3116b2012bc23e99e3833efbd32539bbdd8b84ea54fc17ccf6bR36-R39

Reply:
the API documentation is for a newer version then the one we are using. I couldn't find this (had to dig through the response to implement this)